data/actions/scripts/tools/machete.lua

```diff
@@ -1,8 +1,18 @@
+local jungleGrass = { 2782, 3985, 19433 }
+local wildGrowth = { 1499, 11099 }
+
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
-	if target.itemid == 2782 then
-		target:transform(2781)
+	local targetId = target.itemid
+	if table.contains(jungleGrass, targetId) then
+		target:transform(targetId == 19433 and 19431 or targetId - 1)
```

Comment:
Are you sure about that?

Reply:
yes
![asdasd](https://user-images.githubusercontent.com/4684880/75508080-afba3e00-59c1-11ea-8ca0-fabb9d9d0782.gif)

Reply:
You are right, this is correct but not very readable.
Can you simply use a table of mapping what item is transformed to what?
Would make it more noob-friendly.

Reply:
okay, see it now